### DISCUSSION TITLE
Tableau dans un formulaire

### DISCUSSION THREAD

**Discussion by ChrisThe01** [MTESSDev/FRW#213] [Annonces]
Est-il possible d'insérer un tableau dans un formulaire ? Ce tableau aura quelques lignes avec un entête et une ligne totale. Voici un exemple :
![image](https://github.com/MTESSDev/FRW/assets/129303869/1e2e70d3-1df6-49ef-870c-1183805001df)

**Comment by loima02** [comment]
Bonjour,

Vous pouvez insérer un tableau HTML à l'aide d'un composant s'affichage HTML.

Réf: https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=sectionStyle

**Comment by ChrisThe01** [comment]
Bonjour,

J'ai oublié de mentionner que les montants dans le tableau sont saisissable (sauf le total bien sur). Au fur et mesure, le total s'affiche à jour.

**Comment by j0ber** [comment]
Je ne suis pas certain que l'utilisation d'un tableau soit appropriée pour ce besoin. De toute façon ce n'est pas possible actuellement de mettre un champ dans un tableau.

Un groupe avec 3 champs serait parfait.

Je vais ajouter un exemple dans le P700, je vous reviens avec l'url.

**Comment by j0ber** [comment]
Vous pouvez voir un exemple de ce que j'ai mentionné précédemment ici : https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=sectionInjecterJs   voir exemple 3

**Comment by ChrisThe01** [comment]
Merci ! Je vais essayer ça.

**Comment by ChrisThe01** [comment]
Merci beaucoup pour la belle exemple ! Ça marche à merveille.
Bonne journée,
